presto-docs/src/main/sphinx/security/cli.rst

```diff
@@ -0,0 +1,118 @@
+===========================
+CLI Kerberos Authentication
+===========================
+
+The Presto :doc:`/installation/cli` can connect to a :doc:`Presto coordinator
+</security/server>` that has Kerberos authentication enabled.
+
+Environment Configuration
+-------------------------
+
+.. |subject_node| replace:: client
+.. include:: kerberos-services.fragment
+.. include:: kerberos-configuration.fragment
+
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The cli and coordinator require different principals. The fragment is dead code because of that (good catch). The server.rst file should have similar flavor text but different principals listed in the examples.